### DISCUSSION TITLE
Feature request: An option/config to echo dry-run commands in wet runs

### DISCUSSION THREAD

**Discussion by jiangyilism** [topgrade-rs/topgrade#597] [Ideas]
Currently if we want to check/view what commands topgrade executes we run it with `--dry-run` .
Please consider adding an option/config to print each command like `--dry-run` while doing wet runs.

`--verbose` works but it prints too much. It would be great if we have `--verbose` be less verbose than `--log-filter=debug`. In my opinion `--verbose` is for users while `debug` is more for developers. But that is another story...

If it is not worth adding a new command, then maybe add this behavior to `--log-filter=info`. But personally, I prefer another new log level (e.g. `--log-filter=echo`) since `info` is still too verbose for me.

**Comment by SteveLauC** [comment]
Move this to #598
